src/actions/email.js

```diff
@@ -70,7 +70,9 @@ export function sendMessageToSupport(client, message, t) {
   }
 }
 
-export function sendDeleteAccountRequest(client, subject, message) {
+export function sendDeleteAccountMail(client, subject, message) {
+  if (!message) return
```

Comment:
why not throwing error in that case?

I don't see the point of acting as it's ok, but no mail has been sent